Chart Custom Styles: add PieChart guide (#5016)
DevExpress/devextreme-documentation: concepts/05 UI Components/PieChart/10 Series/30 Customize Appearance.md

> To change series color, use one of the following options:

Reviewer: ```suggestion
To change the color of a series, use one of the following options:
```

> To change series color, use one of the following options:

Reviewer: ```suggestion
To change the color of a series, use one of the following options:
```

> - Specify a color [palette](/concepts/60%20Themes%20and%20Styles/20%20SVG-Based%20Components%20Customization/10%20Palettes/00%20Palettes.md '/Documentation/Guide/Themes_and_Styles/SVG-Based_Components_Customization/#Palettes') for the component.

Reviewer: ```suggestion
- Specify the color [palette](/concepts/60%20Themes%20and%20Styles/20%20SVG-Based%20Components%20Customization/10%20Palettes/00%20Palettes.md '/Documentation/Guide/Themes_and_Styles/SVG-Based_Components_Customization/#Palettes') for the component.
```

> - Specify a [color](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/series/#color) for a series.

Reviewer: ```suggestion
- Specify the [color](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/series/#color) for a series.
```

> - Specify a [color](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/commonSeriesSettings/#color) for all series.

Reviewer: ```suggestion
- Specify the [color](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/commonSeriesSettings/#color) for all series.
```

> Besides color change, you can add a custom pattern or gradient fill to the series.

Reviewer: ```suggestion
You can also add a custom pattern or gradient fill to the series.
```
